deploy/grafana-dashboard/availability_row.py

```diff
@@ -0,0 +1,51 @@
+from grafanalib.core import RowPanel, BarGauge, GridPos, TimeSeries, ORIENTATION_VERTICAL, \
+    GAUGE_DISPLAY_MODE_BASIC
+from grafanalib.influxdb import InfluxDBTarget
+
+from util import read_query
+
+prewarming_pool_size = BarGauge(
+    title="Prewarming Pool Size",
+    dataSource='Poseidon',
+    targets=[InfluxDBTarget(query=read_query("prewarming-pool-size"))],
+    gridPos=GridPos(h=10, w=11, x=0, y=3),
+    allValues=True,
+    orientation=ORIENTATION_VERTICAL,
+    displayMode=GAUGE_DISPLAY_MODE_BASIC,
+    max=None,
+)
+
+idle_runner = TimeSeries(
+    title="Idle Runner",
```

Comment:
I was wondering whether we should allow more fine granular data points here (similar to the current dashboard version) to account for smaller differences.

Reply:
The difference to the original dashboard comes from grafanalib setting a default value for `maxDataPoints`. Before, we have not set a value and so Grafana determines one automatically based on the panel width.
For now, I have adjusted this dashboard to use the dynamic value again.